src/core/types/api/file.ts

```diff
@@ -55,6 +57,21 @@ export type FileAniDBType = {
   Updated: string;
 };
 
+export type FileAVDumpType = {
+  Status: 'Queued' | null;
+  LastDumpedAt: string | null;
+  LastVersion: string | null;
+} | {
+  Status: 'Running' | null;
+  Progress: number;
+  SucceededCreqCount: number;
+  FailedCreqCount: number;
+  PendingCreqCount: number;
+  StartedAt: string;
+  LastDumpedAt: string | null;
+  LastVersion: string | null;
+};
+
```

Comment:
Why two types?
Combine them and keep the extra ones optional.
Because this will cause issues

Reply:
So we know which fields are populated when.

Reply:
Examples;
![image](https://github.com/ShokoAnime/Shoko-WebUI/assets/7761729/b61f45c0-ea3f-45b0-8591-2029977704a3)
![image](https://github.com/ShokoAnime/Shoko-WebUI/assets/7761729/e99bd0d2-835c-4313-b120-b626d4e8a64f)

Reply:
Fudge. I spot an error.